src/drivers/imx/interrupt.c

```diff
@@ -3,26 +3,413 @@
 // Copyright 2019 NXP
 //
 // Author: Daniel Baluta <daniel.baluta@nxp.com>
+// Author: Paul Olaru <paul.olaru@nxp.com>
 
 #include <sof/drivers/interrupt.h>
+#include <sof/lib/io.h>
+#include <inttypes.h>
```

Comment:
Where do you use anything from this header?

Reply:
Not sure about io.h but I do use PRIx64 from <inttypes.h>. I will re-add io.h if it fails the build.

Reply:
You were referring to inttypes indeed. Re-added io.h, coming up next push.

Reply:
io is OK, you are using it.
And passing build won't give you any information, because it can be included in different header, but it's a good practice to include every header you're using.